src/aliceVision/image/io.hpp

```diff
@@ -79,22 +97,23 @@ void readImageMetadata(const std::string& path, int& width, int& height, std::ma
  * @brief read an image with a given path and buffer
  * @param[in] path The given path to the image
  * @param[out] image The output image buffer
+ * @param[in] image color space
  */
-void readImage(const std::string& path, Image<float>& image);
-void readImage(const std::string& path, Image<unsigned char>& image);
-void readImage(const std::string& path, Image<RGBAColor>& image);
-void readImage(const std::string& path, Image<RGBfColor>& image);
-void readImage(const std::string& path, Image<RGBColor>& image);
+void readImage(const std::string& path, Image<float>& image, EImageColorSpace imageColorSpace = EImageColorSpace::NO_CONVERSION);
```

Comment:
We should remove the default value of `imageColorSpace`, so it's always specified explicitly.
Same for `writeImage`.